docs/components/Layout.less

```diff
@@ -57,12 +71,17 @@ a {
   &:hover,
   &:focus,
   &:active {
+    .text--decoration-none;
     outline: 0;
-    text-decoration: none;
   }
 
   &:hover,
   &:focus {
     .shade(color, @primary_blue, 2);
   }
 }
+
+code {
+  .text--monospace;
+  background-color: @neutral_off_white;
+}
```

Comment:
nit: missing newline (might need a whitespace correction plugin for your IDE to fix on save)